MkDocs/apm_8.8/docs/matrix.md

```diff
@@ -0,0 +1,100 @@
+# Asset Matrix
+
+Maximo HP Utilities includes matrix view for determing the distribution of assets based on scores and then take action to avoid imminent failures.
+
+In this exercise the Reliability Engineer uses HP Utilities to:
+	
+1. [View Matrix for Criticality and End of life, Criticality and Risk, Criticality and Health](#view-matrix-for-criticality-and-end-of-life-criticality-and-risk-criticality-and-health)
+2. [View the drilled in assets list and take action](#view-the-drilled-in-assets-list-for-one-distribution)
+3. [Config Matrix version](#config-matrix-version)
+
+
+## Pre-requisites 
+
+This lab requires the following
+
```

Comment:
Before we can go live with this Risk matrix md   I think we need someone to go through the v8.7 APM lab and update/ test for MAS v8.8

Reply:
Do you mean someone go through 8.7 content, and copy 8.7 content to 8.8 folder?